include/groonga/groonga.h

```diff
@@ -1520,6 +1520,28 @@ grn_obj_path(grn_ctx *ctx, grn_obj *obj);
 GRN_API int
 grn_obj_name(grn_ctx *ctx, grn_obj *obj, char *namebuf, int buf_size);
 
+/**
+ * \brief Return a column's name by storing it in `namebuf`.
+ *
+ * If the column has a name and `buf_size` is greater than or equal to
+ * the length of the name, the name is copied into `namebuf`.
+ *
+ * If the length of the name exceeds `buf_size`, the name is not copied
+ * into `namebuf`, but the length of the name is still returned.
+ *
+ * The maximum possible length of the name is limited by
+ * \ref GRN_TABLE_MAX_KEY_SIZE.
+ *
+ * If the `obj` is an accessor, it returns the accessor's name instead.
```

Comment:
```suggestion
 * If the `obj` is an accessor, it returns the accessor's name instead.
 * Accessor's name uses `${PATH1}.${PATH2}.${...}.${PATHN}` format.
 * For example:
 * * `_key` for accessing a key of the target object.
 * * `nested.column.value` for accessing `nested`参照カラムの先の`column`参照カラムの先の`value`カラムの値
```

アクセサーのフォーマットってどこかに説明があったっけ？あるならそこへのリンクでいいかも。

Reply:
現状はないので、こちらに追記しようと思います！

Reply:
https://github.com/groonga/groonga/blob/499d60f4ad1b9dd15b48b7d3383718897cc446eb/include/groonga/groonga.h#L1023 に微妙に例がありますね。。。

Reply:
同じ部分に説明もあるので、一旦そちらのリンクにしたほうが良さそうですね。
少し物足りなさそうでしたら、具体例もつぎのPRで`grn_obj_column()`に追記しようと思います。
https://github.com/groonga/groonga/blob/499d60f4ad1b9dd15b48b7d3383718897cc446eb/include/groonga/groonga.h#L1011-L1013

Reply:
fix: [72fb307](https://github.com/groonga/groonga/pull/2084/commits/72fb307fb3e065f17f7cc661ecbca40571f0ba46)